src/resolvers/index.ts

```diff
@@ -19,4 +30,15 @@ export const resolvers: Resolvers = {
   Organization,
   Query,
   Subscription,
+
+  // graphql-scalar resolver
+  Date: DateResolver,
+  DateTime: DateTimeResolver,
```

Comment:
Please try to place items lexographically wherever possible. It's makes it easier to navigate and takes less mental overhead to make sense of things.

Reply:
All the items are placed loexographically already. I wanted to keep the graphql-scalar resolvers and our imported resolvers separate, that's why I didn't merge them. 

Should I mix both of them, and then sort them lexographically?

Reply:
Hmm, let it be no problem.